2023/1/trebuchet.input

```diff
@@ -0,0 +1,7 @@
+two1nine
```

Comment:
For checking into the repo, we usually want the example input, not yours specifically.  Your code should work with either one.

Reply:
This is the example input, at least for part 2. Should we commit our solutions for part1 or part2?

Reply:
Ah.  We usually do both.

Reply:
The readme for the repo also says to name the files `-part1` and `-part2` when there are different files for each part.  So your `trebuchet.input` should be `trebuchet-part2.input`, and you'll need to add `trebuchet-part1.input` along with the code for part one (code for both parts can be in the same file - again, check previous year's files... some people did all-in-one, while others did separate files).